gix-blame/src/file/function.rs

```diff
@@ -252,6 +199,24 @@ where
     })
 }
 
+/// The blobs storing the blamed file in `entry` and `parent_entry` are identical which is why
+/// we can pass blame to the parent without further checks.
```

Comment:
The comment refers to this line: `entry.oid == parent_entry.oid`. Since this line is not part of the function, it seems a little off. :-)

Reply:
Thanks, this is fixed now.